tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/mappingconstructor/MappingConstructorExprTest.java

```diff
@@ -212,8 +212,9 @@ public void testSpreadOpFieldCodeAnalysisNegative() {
     public void testSpreadOpFieldConstantAnalysisNegative() {
         CompileResult result = BCompileUtil.compile(
                 "test-src/expressions/mappingconstructor/spread_op_field_constant_analysis_negative.bal");
-        Assert.assertEquals(result.getErrorCount(), 1);
-        validateError(result, 0, "expression is not a constant expression", 19, 47);
+        int i = 0;
+        validateError(result, i++, "undefined symbol 'constFloatMap'", 19, 47);
```

Comment:
This error message is wrong. Here, constFloatMap is defined, but not a constant symbol. That is different from undefined symbol.

```ballerina
map<float> constFloatMap = {z: 1.0};

const map<float> constFloatMap2 = {a: 2.0, ...constFloatMap};
```

Reply:
I will update that

Reply:
Updated c64fd4e460acdf2c3f75e0794b4064834619cc8d